src/containers/AdminDashboard/components/Register10DlcNoticeCard.tsx

```diff
@@ -0,0 +1,64 @@
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardHeader from "@material-ui/core/CardHeader";
+import OpenInNew from "@material-ui/icons/OpenInNew";
+import Warning from "@material-ui/icons/Warning";
+import type { Register10DlcNotice } from "@spoke/spoke-codegen";
+import React from "react";
+
+import BrandRegistration10DlcNoticeText from "../../../components/NoticeText/BrandRegistration10DlcNoticeText";
+import CampaignRegistration10DlcNoticeText from "../../../components/NoticeText/CampaignRegistration10DlcNoticeText";
+
+interface Register10DlcNoticeCardProps extends Register10DlcNotice {
+  __typename?: string;
+}
+
+const Register10DlcNoticeCard: React.FC<Register10DlcNoticeCardProps> = (
+  props
+) => {
+  const isBrandNotice = props.__typename === "Register10DlcBrandNotice";
+  return (
+    <Card variant="outlined" style={{ marginBottom: "2em" }}>
+      <CardHeader
+        title={
+          isBrandNotice
+            ? "Register for 10DLC to reduce your sending costs"
+            : "Incomplete 10DLC Registration"
+        }
+        avatar={<Warning color="error" />}
+      />
+      <CardContent>
+        {isBrandNotice ? (
+          <BrandRegistration10DlcNoticeText />
+        ) : (
+          <CampaignRegistration10DlcNoticeText />
+        )}
+
+        {props.tcrRegistrationUrl === null && (
```

Comment:
nit question: do you intend to only check for `null` here, or are you trying to check for nullish values (`null` or `undefined`)?

Reply:
Just null works yeah. Helped draw my attention to some related stuff that needed to be changed though 😃